aws/secrets.mk

```diff
@@ -9,14 +10,22 @@ CMD_SERVICE_SECRETS_PUSH = @ (echo $(foreach item, $(SERVICE_SECRETS), \
 			cat $(SERVICE_SECRETS_FILE) | $(JQ) -r '.$(item)' \
 		)" --type SecureString --overwrite && \
 		$(AWS) --profile=$(AWS_PROFILE) ssm add-tags-to-resource --resource-type "Parameter" --resource-id "/$(ENV)/$(SVC)/$(item)" \
-		--tags "Key=Application,Value=$(SVC)" "Key=EnvVarName,Value=$(item)" \
-	)) > /dev/null ) && echo "\033[32m[OK]\033[0m $(SVC) secrets upload" || echo "\033[31m[ERROR]\033[0m $(SVC) secrets upload"
+		--tags "Key=Application,Value=$(SVC)" "Key=EnvVarName,Value=$(item)" || \
+		echo "\033[31m[ERROR]\033[0m /$(ENV)/$(SVC)/* secrets upload")) > /dev/null ) && echo "\033[32m[OK]\033[0m $(SVC) secrets upload" || echo "\033[31m[ERROR]\033[0m $(SVC) secrets upload"
 
 CMD_SERVICE_SECRETS_DELETE = @ (echo $(foreach item, $(shell $(AWS) --profile=$(AWS_PROFILE) ssm get-parameters-by-path \
 		--path "/$(ENV)/$(SVC)" --query "Parameters[*].Name" --recursive | $(JQ) -e -r '. | to_entries[] | .value' ), \
 		$(shell $(AWS) --profile=$(AWS_PROFILE) ssm delete-parameter --name $(item))) > /dev/null ) && \
 		echo "\033[32m[OK]\033[0m /$(ENV)/$(SVC)/* secrets deleted" || echo "\033[31m[ERROR]\033[0m /$(ENV)/$(SVC)/* secrets deletion"
 
+CMD_SERVICE_ALL_SECRET_KEYS = $(foreach item, $(shell aws ssm --profile=$(AWS_PROFILE) get-parameters-by-path \
```

Comment:
@igorkotof let's update  `aws` to `$(AWS) --profile` ?

Reply:
@AutomationD fixed